### ISSUE TITLE
Mixed content (such as GIFs) doesn't show

### ISSUE THREAD

**Issue by lloeki** [lloeki/matterfront#39]
Log in console:

```
Mattermost:  Mixed Content: The page at 'https://mattermost.example.com/foo/channels/sandbox' was loaded over HTTPS, but requested an insecure image 'http://i3.kym-cdn.com/photos/images/newsfeed/000/288/648/776.gif'. This request has been blocked; the content must be served over HTTPS.
```

**Comment by LongLiveCHIEF** [comment]
There were a lot of new features with Electron `v0.36` rolled out that might allow us to implement a fix for this without disabling security.  I'm going to dig around a bit and see if I can get something working.

**Comment by geekytime** [comment]
If you set the proxy settings via chrome-args, this might work now.

**Comment by jonykalavera** [comment]
chrome has a --allow-running-insecure-content flag. sadly electron doesn't seem to support it https://github.com/atom/electron/blob/master/docs/api/chrome-command-line-switches.md

**Comment by LongLiveCHIEF** [comment]
Actually it does. See https://github.com/HackerHappyHour/matterfront/pull/41

We have other ways to support the proxy in progress, aND we now also have support for the --no-proxy flag.

**Comment by liamdawson** [comment]
Progress?

**Comment by LongLiveCHIEF** [comment]
Nope!  See [comments here](https://github.com/HackerHappyHour/matterfront/issues/86#issuecomment-187794039).
